asv_bench/benchmarks/tslibs/strftime.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
+
+import pandas as pd
+from pandas import offsets
+
+
+class DatetimeStrftime:
+    fname = "__test__.csv"
```

Comment:
are these used anywhere?

Reply:
I don't know what this is for, I just copied another bench..

Reply:
if its not used then remove it